pythran/backend.py

```diff
@@ -441,7 +442,7 @@ def make_const_function_declaration(rtype, name, ftypes, fargs,
                 Cxx.generator_state_holder,
                 " ".join("case {0}: goto {1};".format(num, where)
                          for (num, where) in sorted(
-                             self.yields.itervalues(),
+                             iter(self.yields.values()),
```

Comment:
On Thu, Jul 21, 2016 at 12:03:07PM -0700, pbrunet wrote:
> > -                             self.yields.itervalues(),
> > +                             iter(self.yields.values()),
> 
> Why not just value?

2to3 decided this, cannot tell you why...